src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/blob/InputScannableItem.java

```diff
@@ -15,7 +17,7 @@
     public final String manualIntervention;
     public final String nextAction;
     public final Timestamp nextActionDate;
-    public final String ocrData;
+    public final Map<String, String> ocrData;
```

Comment:
OCR data can be either deserialised straight to a map, like here, or deserialised to string and converted later. The second solution would make the process a bit trickier, so I went with this one, but maybe you've got some thought?

Reply:
Straight to `Map` is so much acceptable